src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx

```diff
@@ -112,18 +119,18 @@ export const IsomerDetailGroupView = ({
   )
   return (
     <BlockWrapper
-      name="Accordion grid"
+      name="Accordion"
       isSelected={selected}
       childButtons={otherButtons}
       padding-top="0.5rem"
       padding-bottom="0.5rem"
     >
       <Box
         borderColor="base.divider.strong"
-        borderWidth="1px"
-        mt="1rem"
         h="100%"
         w="100%"
+        borderTop="1px solid #d4d4d4"
+        borderBottom="1px solid #d4d4d4"
```

Comment:
should we use colour tokens here

Reply:
sadly we dont have a token for this yet.
@sehyunidaaa may i get your help in this ya

Reply:
ahh this is cos i used the colors from the template. my bad!

can use `divider/strong` [from OGPDS please](https://github.com/opengovsg/design-tokens/blob/09eeb37eaca44f0cb56e64d992cdee75b34e7661/tokens.json#L640) thank you